.chalice/config.json

```diff
@@ -7,10 +7,10 @@
          "ES_REGION": "us-west-2"
       }
     },
-    "staging": {
-      "api_gateway_stage": "staging",
+    "commonsstaging": {
+      "api_gateway_stage": "commonsstaging",
       "environment_variables": {
-         "ES_HOST": "search-dss-azul-commons-lx3ltgewjw5wiw2yrxftoqr7jy.us-west-2.es.amazonaws.com",
+         "ES_HOST": "search-dss-index-commonsstaging-umpfdpphkpmtiwkuojfyjuko6a.us-west-2.es.amazonaws.com",
```

Comment:
Is this going to change often? Should we make a permanent DNS entry for our staging ES instance?